### PR TITLE
experimenting with menu changes

### PR DESCRIPTION
**Pull request by boazsender**
This is not really meant to be merged, but more as a conversation about what the ui nav items should be labeled as:

image

* I think "cohort management" makes more sense than "facilitator" at this point.
* I think "Moments" makes more sense than "curated"
* I think "researcher" needs to be merged in as a sub-view of "cohorts".

Does this make sense?

### CODE REVIEW

**Comment by rwaldron** [comment]
Managing cohorts is only one thing that a Facilitator might do, which is why Facilitators have a Facilitator Dashboard. Also, this "/cohort/" path conflicts with a "/cohort/:id" path that I have in the feature that I'm developing and that should take precedence over unplanned changes.

File: client/routes/Navigation.jsx [hunk 1]
```diff
@@ -36,12 +36,12 @@ const Navigation = () => {
             </Button>
             {menuExpanded && (
                 <Menu id="nav" className="nav" stackable>
-                    <Menu.Item>
+                    <Menu.Item className="logo">
```
Comment (rwaldron) [inline client/routes/Navigation.jsx]: Where is this defined?

File: client/routes/Navigation.jsx [hunk 2]
```diff
@@ -36,12 +36,12 @@ const Navigation = () => {
...
                         <NavLink to="/">Teacher Moments</NavLink>
                     </Menu.Item>
 
                     <Menu.Menu>
-                        <Dropdown simple item text="Curated">
+                        <Dropdown simple item text="Moments">
```
Comment (rwaldron) [inline client/routes/Navigation.jsx]: This drop down is supposed to move under the item presently called "Teacher Moments", but we were waiting until we had a final word on the corner logo. This menu item needs to merge with that item, not be renamed.

File: client/routes/Navigation.jsx [hunk 3]
```diff
@@ -60,13 +60,13 @@ const Navigation = () => {
                                 </NavLink>
                             </Menu.Item>
                             <Menu.Item>
-                                <NavLink to="/facilitator">Facilitator</NavLink>
+                                <NavLink to="/cohorts">Cohorts Management</NavLink>
```
Comment (rwaldron) [inline client/routes/Navigation.jsx]: Please revert this change

File: client/routes/Navigation.jsx [hunk 4]
```diff
@@ -60,13 +60,13 @@ const Navigation = () => {
...
                             </Menu.Item>
                             <Menu.Item>
-                                <NavLink to="/researcher">Researcher</NavLink>
+                                <NavLink to="/researcher">Data Export (to be merged with cohorts)</NavLink>
```
Comment (rwaldron) [inline client/routes/Navigation.jsx]: Please revert this change, Researcher view is a **different** class of data: only the consented data can be shown there, where Facilitator can see all data for the cohort they are running.

File: client/routes/Navigation.jsx [hunk 5]
```diff
@@ -60,13 +60,13 @@ const Navigation = () => {
...
                             </Menu.Item>
                             <Menu.Item>
-                                <NavLink to="/admin">Admin</NavLink>
+                                <NavLink to="/account-administration">Account Administration</NavLink>
```
Comment (rwaldron) [inline client/routes/Navigation.jsx]: What is the motivation for this change?

File: client/routes/Routes.jsx [hunk 6]
```diff
@@ -60,8 +60,8 @@ const Routes = () => {
             </InterceptAnonymizableRoute>
             <Route exact path="/editor/new" component={NewScenario} />
             <Route path="/editor/:id" component={Editor} />
-            <Route exact path="/facilitator" component={Facilitator} />
-            <Route exact path="/admin" component={AccountAdmin} />
+            <Route exact path="/cohorts" component={Facilitator} />
```
Comment (rwaldron) [inline client/routes/Routes.jsx]: Revert the change from `/facilitator` to `/cohorts`

**Comment by rwaldron** [comment]
Please fix the lint nits

File: client/routes/Navigation.jsx (hunk 2 above)
Comment (boazsender) [reply client/routes/Navigation.jsx]: sgtm

File: client/routes/Navigation.jsx (hunk 1 above)
Comment (boazsender) [reply client/routes/Navigation.jsx]: whoops. you can disregard this.

File: client/routes/Navigation.jsx (hunk 5 above)
Comment (boazsender) [reply client/routes/Navigation.jsx]: Consistency with my other suggestions.

**Comment by boazsender** [comment]
> Managing cohorts is only one thing that a Facilitator might do, which is why Facilitators have a Facilitator Dashboard. Also, this "/cohort/" path conflicts with a "/cohort/:id" path that I have in the feature that I'm developing and that should take precedence over unplanned changes.

File: client/routes/Navigation.jsx (hunk 4 above)
Comment (boazsender) [reply client/routes/Navigation.jsx]: I think that researchers and facilitators should actually see the same view, just with different scopes of data. Even if someone were a researcher and a facilitator, this should work in one view which when loaded:

* if I am a facilitator I see all the cohorts I have created with all of the associated run-response data for each one
* if I am a researcher I see the same thing for my cohorts plus consent-only data for all other cohorts.

Comment (rwaldron) [reply client/routes/Navigation.jsx]: I agree with your points here, thanks for clarifying.

**Comment by rwaldron** [comment]
After more thought and making more progress on cohorts, I think you've convinced me of the name change.
